static/global.css

```diff
@@ -44,7 +71,7 @@ button {
   border: none;
   cursor: pointer;
   letter-spacing: 0;
-  font-family: interstate;
+  font-family: inherit;
   font-weight: 700;
```

Comment:
`font-weight: bold;`

Reply:
This shouldn't be bold actually! Would prefer the button to inherit the font family of the container